include/oneapi/tbb/concurrent_vector.h

```diff
@@ -565,6 +565,14 @@ class concurrent_vector
         return new_segment_table;
     }
 
+    void deallocate_long_table(segment_table_type table) {
+        auto& alloc = base_type::get_allocator();
+        for (size_type seg_idx = 0; seg_idx < this->pointers_per_long_table; ++seg_idx) {
+            segment_table_allocator_traits::destroy(alloc, &table[seg_idx]);
+        }
+        segment_table_allocator_traits::deallocate(alloc, table, this->pointers_per_long_table);
```

Comment:
Can we introduce this code as an extra method in `segment_table` and reuse in `segment_table::clear_table`?

Reply:
Good idea! Applied.